package.json

```diff
@@ -24,6 +24,8 @@
     "karma-spec-reporter": "0.0.13",
     "mocha": "~1.15.1",
     "phantomjs": ">= 1.9.7-1",
+    "phantomjs-prebuilt": ">= 1.9",
```

Comment:
It seems that we can. :)

Not only that but it removes the requirement to manually add the `-prebuilt` package too. \o/